owncloud/core
# [Not merge] Remove patchwork library

## Description
Remove the patchwork library because it won't be used anymore

## Related Issue
Requested in https://github.com/owncloud/core/pull/29500#issuecomment-342604553

## Motivation and Context


## How Has This Been Tested?
Manually tested:

* ✅  Having the php-intl package installed and a working ownCloud installation, I've created some folders with utf8 names and upload several files also with utf8 name. No problems detected.

    The behaviour seems to be the same with and without the library, which includes some chars drawn as fallback symbols in some situations (very likely browser related, but anyway patchwork won't fix the issue)
* ⚪️  Removing the php-intl package will crash ownCloud (kind of expected):

    ```
    {"reqId":"HJIE7sQ6C9tMY6AmcmSL","level":3,"time":"2017-11-08T09:28:06+00:00","remoteAddr":"10.0.2.4","user":"admin","app":"index","method":"GET","url":"\/index.php\/apps\/files\/","message":"Exception: {\"Exception\":\"Error\",\"Message\":\"Class 'Normalizer' not found\",\"Code\":0,\"Trace\":\"#0 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Files\\\/Filesystem.php(864): OC_Util::normalizeUnicode('\\\/admin\\\/files_ex...')\\n#1 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Files\\\/View.php(1099): OC\\\\Files\\\\Filesystem::normalizePath('\\\/admin\\\/files_ex...')\\n#2 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Files\\\/View.php(380): OC\\\\Files\\\\View->basicOperation('is_dir', '\\\/admin\\\/files_ex...')\\n#3 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Security\\\/CertificateManager.php(76): OC\\\\Files\\\\View->is_dir('\\\/admin\\\/files_ex...')\\n#4 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Http\\\/Client\\\/Client.php(66): OC\\\\Security\\\\CertificateManager->listCertificates()\\n#5 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Http\\\/Client\\\/Client.php(135): OC\\\\Http\\\\Client\\\\Client->setDefaultOptions()\\n#6 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Updater\\\/VersionCheck.php(111): OC\\\\Http\\\\Client\\\\Client->get('https:\\\/\\\/updates...')\\n#7 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/Updater\\\/VersionCheck.php(81): OC\\\\Updater\\\\VersionCheck->getUrlContent('https:\\\/\\\/updates...')\\n#8 \\\/opt\\\/owncloud\\\/apps\\\/updatenotification\\\/lib\\\/UpdateChecker.php(42): OC\\\\Updater\\\\VersionCheck->check()\\n#9 \\\/opt\\\/owncloud\\\/apps\\\/updatenotification\\\/appinfo\\\/app.php(36): OCA\\\\UpdateNotification\\\\UpdateChecker->getUpdateState()\\n#10 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/app.php(223): require_once('\\\/opt\\\/owncloud\\\/a...')\\n#11 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/app.php(174): OC_App::requireAppFile('updatenotificat...')\\n#12 \\\/opt\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/app.php(128): OC_App::loadApp('updatenotificat...')\\n#13 \\\/opt\\\/owncloud\\\/lib\\\/base.php(913): OC_App::loadApps()\\n#14 \\\/opt\\\/owncloud\\\/index.php(56): OC::handleRequest()\\n#15 {main}\",\"File\":\"\\\/opt\\\/owncloud\\\/lib\\\/private\\\/legacy\\\/util.php\",\"Line\":1359}"}
    ```
* ~After reinstalling the php-intl package, there has been some problems with files having utf8 chars, not sure if some weird locale was set at some point. The "Setting locale to ... failed" error message appeared.~ https://github.com/owncloud/core/pull/29508/commits/1389f290fe2ea445b831d308738723270258ca75 seems to solve the problem
* 🔴  ~Upgrade hasn't been tested yet, but it seems we should prevent the upgrade somehow if the php-intl package isn't installed.~ Check https://github.com/owncloud/core/pull/29508#issuecomment-342807566

## Screenshots (if appropriate):

## Types of changes


## Checklist:



Note that some of the sabre urls in the composer.lock file have changed.
